### PR TITLE
[WIP] Document where config comes from

### PR DESCRIPTION
**Pull request by yuvipanda**
Commit to help start discussion

### CODE REVIEW

**Comment by yuvipanda** [comment]
I've also opened https://github.com/yuvipanda/jupyterhub-configurator/issues/8 to talk more about the configurator

**Review by damianavila — APPROVED** [review]
I think this a great start!
We can go from here in subsequent PRs if that is actually necessary.

**Review by damianavila — COMMENTED** [review]
Actually, I approved more quickly than it should be.
I think there is a missing reference in the index.md, otherwise, I do not see this new page in the side column :wink:

File: docs/topic/config.md
Comment (choldgraf) [inline docs/topic/config.md:9]:
```suggestion
| [`hub-templates/base-hub/values.yaml`](https://github.com/2i2c-org/pilot-hubs/blob/master/hub-templates/base-hub/values.yaml) | 2i2c engineers | Common to all hubs run from this repo |
```

File: docs/topic/config.md
Comment (choldgraf) [inline docs/topic/config.md:10]:
```suggestion
| [`deploy.py`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py) | 2i2c engineers | Programmatic overrides for each type of hub, particularly around Auth0 integration |
```

File: docs/topic/config.md
Comment (choldgraf) [inline docs/topic/config.md:11]:
```suggestion
| [`hubs.yaml`](https://github.com/2i2c-org/pilot-hubs/blob/master/hubs.yaml) | 2i2c engineers | Specific to each hub |
```

**Review by choldgraf — COMMENTED** [review]
This looks good to me - just added a couple of links in there so people can click-through easily for reference.

Once this is linked in the toctree I think this is good to merge! (assuming it is factually correct and complete, I'll defer to @yuvipanda on that)

**Pull request closed without merging**
